tests/private-tcp-active-active/proxy.tf

```diff
@@ -20,17 +20,6 @@ resource "aws_security_group_rule" "proxy_ingress_mitmproxy" {
   security_group_id = aws_security_group.proxy.id
 }
 
-resource "aws_security_group_rule" "proxy_ingress_ssh" {
```

Comment:
[This test](https://github.com/hashicorp/terraform-aws-terraform-enterprise/runs/2843576320?check_suite_focus=true) passed when this rule was removed.